python/ml_wrappers/model/predictions_wrapper.py

```diff
@@ -0,0 +1,199 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+"""Defines classes to wrap the training/test data and the corresponding predictions from the model."""
+
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+
+TARGET = 'target'
+
+
+class ModelWrapperPredictions:
```

Comment:
nit: maybe PredictionsModelWrapper is a better name for the class than ModelWrapperPredictions?

Reply:
Done. Thanks for the suggestion.